matsim/pom.xml

```diff
@@ -5,7 +5,7 @@
 	<groupId>org.matsim</groupId>
 	<artifactId>matsim</artifactId>
 	<name>MATSim</name>
-  <version>0.10.1-beam-4</version>
+  <version>0.10.1-beam-5</version>
```

Comment:
do we have to increase this with every commit? problem: we have to change gradle build file on master as well each time...

Reply:
Yes, correct. If we want to have an ability to rollback to previous version we should have versioning. Also most of the time there should be a reason to update MATSim and make the changes visible in `beam` (bump-up `build.gradle`) - so it still make sense for me to have versions.